rolling/distribution.yaml

```diff
@@ -917,6 +917,25 @@ repositories:
       url: https://github.com/carologistics/clips_vendor.git
       version: main
     status: maintained
+  cob_common:
+    doc:
+      type: git
+      url: https://github.com/4am-robotics/cob_common.git
+      version: foxy
+    release:
+      packages:
+      - cob_actions
+      - cob_msgs
+      - cob_srvs
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/4am-robotics/cob_common-release.git
```

Comment:
In order to release into Rolling, the release repository must be in the `ros2-gbp` organization.  Please see https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/CONTRIBUTING.md on how to do this.

Reply:
Thank you for the hint, I wasn't aware of this.
- Issue: https://github.com/ros2-gbp/ros2-gbp-github-org/issues/665
- PR: https://github.com/ros2-gbp/ros2-gbp-github-org/pull/666